src/Tickets/Emails/Provider.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * The main service provider for the Tickets Emails.
+ *
+ * @since   TBD
+ * @package TEC\Tickets\Emails
+ */
+
+namespace TEC\Tickets\Emails;
+
+use tad_DI52_ServiceProvider;
+
+/**
+ * Service provider for the Tickets Emails.
+ *
+ * @since   TBD
+ * @package TEC\Tickets\Emails
+ */
+class Provider extends tad_DI52_ServiceProvider {
+
+	/**
+	 * Register the provider singletons.
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+
+		// If not enabled, do not load Tickets Emails system.
+		if ( ! tec_tickets_emails_is_enabled() ) {
+			return;
+		}
+
+		$this->container->register( Emails_Tab::class );
+
```

Comment:
Let's just register the plain Assets and Hooks class here.

Reply:
Like this: https://github.com/the-events-calendar/event-tickets-plus/blob/085ad53b55594de1afb1701860b53815b3fd5449/src/Tribe/Manual_Attendees/Service_Provider.php#L21

Reply:
We're keeping the same organizational structure as we have in Tickets Commerce, here. All asset and hook related things can go in their own method.